src/JSInterop/Microsoft.JSInterop/src/IJSStreamReference.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.JSInterop
+{
+    /// <summary>
+    /// Represents a reference to JavaScript data to be consumed through a <see cref="Stream"/>.
+    /// </summary>
+    public interface IJSStreamReference : IAsyncDisposable
+    {
+        /// <summary>
+        /// Length of the <see cref="Stream"/> provided by JavaScript.
+        /// </summary>
+        public long Length { get; }
```

Comment:
Is the `public` modifier here intentional? I think adding it makes this a default implementation.

Reply:
Looks like it should be fine to remove, will update in a follow-up PR. Thanks!